WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -19,7 +19,23 @@ final public class InteractiveNotificationsManager: NSObject {
         return UIApplication.shared
     }
 
+    /// Returns the Core Data main context.
+    ///
+    var context: NSManagedObjectContext {
+        return ContextManager.sharedInstance().mainContext
+    }
 
+    /// Returns a CommentService instance.
+    ///
+    var commentService: CommentService {
+        return CommentService(managedObjectContext: context)
+    }
+
+    /// Returns a NotificationSyncMediator instance.
+    ///
+    var notificationSyncMediator: NotificationSyncMediator? {
+       return NotificationSyncMediator.init()
```

Comment:
No need to explicitly call `init()`. `NotificationSyncMediator()` should suffice 😄